packages/one-click-importer/src/components/OneClickImporter.vue

```diff
@@ -97,6 +97,10 @@
     padding-top: 0.5em;
     font-size: smaller;
   }
+
+  #file-input {
```

Comment:
Could you do a class ?, for example '.mg-file-input'  , that way we can keep classes for styling and use id's for linking to js and form stuff.  and maybe add a comment line explaining this is a work around ?